docs/Initial-Setup/Setting-up-your-Manager-Instance.rst

```diff
@@ -4,6 +4,11 @@ Setting up your Manager Instance
 Launching a "Manager Instance"
 ------------------------------
 
+.. warning::
+    These instructions refer to fields in EC2's new launch instance
+    wizard. Refer to version 1.13 of the documentation for references to the old
```

Comment:
I would prefer this as a link to the 1.13.x docs. Maybe just point to `1.13.4` for now? Or maybe just mark this in an issue that we resolve when we release (so we can get the latest minor release).